features/networking/sctp.feature

```diff
@@ -26,48 +27,45 @@ Feature: SCTP related scenarios
       | ["spec"]["nodeName"]      | <%= cb.workers[1].name %> |
     Then the step should succeed
     And the pod named "sctpclient" becomes ready
-
+   
     # sctpserver pod start to wait for sctp traffic
-    And I wait up to 60 seconds for the steps to pass:
-    """
     When I run the :exec background client command with:
-      | pod              | sctpserver          |
-      | namespace        | <%= project.name %> |
-      | oc_opts_end      |                     |
-      | exec_command     | bash                |
-      | exec_command_arg | -c                  |
-      | exec_command_arg | nc -l 30102 --sctp  |
-    Then the step should succeed
-    """
+      | pod              | sctpserver            |
+      | namespace        | <%= project.name %>   |
+      | oc_opts_end      |                       |
+      | exec_command     | bash                  |
+      | exec_command_arg | -c                    |
+      | exec_command_arg | nc -k -l 30102 --sctp |
 
     # sctpclient pod start to send sctp traffic
     And I wait up to 60 seconds for the steps to pass:
-    """"
+    """
     When I run the :exec client command with:
-      | pod              | sctpclient                                                       |
-      | namespace        | <%= project.name %>                                              |
-      | oc_opts_end      |                                                                  |
-      | exec_command     | bash                                                             |
-      | exec_command_arg | -c                                                               |
-      | exec_command_arg | echo test-openshift \| nc -v <%= cb.serverpod_ip %> 30102 --sctp |
+      | pod              | sctpclient                                                            |
+      | namespace        | <%= project.name %>                                                   |
+      | oc_opts_end      |                                                                       |
+      | exec_command     | bash                                                                  |
+      | exec_command_arg | -c                                                                    |
+      | exec_command_arg | echo test-openshift \| nc -w 5 -v <%= cb.serverpod_ip %> 30102 --sctp |
```

Comment:
We can use 
```
When I execute on the "xxx" pod:
  | blabla |
```

Reply:
@liangxia Your suggestion makes steps more short now, thanks! New test log: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2849/consoleFull